addons/io_scene_gltf2/blender/exp/gltf2_blender_gather_materials.py

```diff
@@ -143,6 +176,13 @@ def __gather_extensions(blender_material, export_settings):
 
     return extensions if extensions else None
 
+def __get_tex_from_socket(blender_shader_socket: bpy.types.NodeSocket):
+    result = gltf2_blender_search_node_tree.from_socket(
+        blender_shader_socket,
+        gltf2_blender_search_node_tree.FilterByType(bpy.types.ShaderNodeTexImage))
+    if not result:
+        return None
+    return result[0]
```

Comment:
Similar to above, since this method is duplicated, could we move it to `gltf2_blender_get.py` and reuse it here and in `gltf2_blender_gather_materials_pbr_metallic_roughness.py`?